### ISSUE TITLE
Tutorial Video Request

### ISSUE THREAD

**Issue by miroccur** [iMokhles/Tweaks-for-SideLoaded-Apps#4]
Hi @iMokhles , first I should say thanks to you, but I'm completely a noob, could you make one tutorial video so that user will know how to play with your project more easily? Thanks.

**Comment by al-muftah** [comment]
yes please we need a tutorial,it seems it's a great job , but i don't know how to install them 

thanks a lot

**Comment by onesentinel** [comment]
If you guys really need some help I prob. could help you somewhere. I'm also in the learning mode, and trying to figure out how things works but I already know at least how most of it works
